### PR TITLE
feat(@desktop/chat): Style gif widget component

### PR DESCRIPTION
**Pull request by alaibe**
This also improve the onload of the gif using the tinyUrl

image
image

### CODE REVIEW

File: ui/shared/status/StatusGifPopup.qml [hunk 1]
```diff
@@ -102,23 +182,23 @@ Popup {
             Row {
                 anchors.horizontalCenter: parent.horizontalCenter
                 anchors.verticalCenter: parent.verticalCenter
-                spacing: 4
+                spacing: 8
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifPopup.qml:185]:
```suggestion
                spacing: Style.current.halfPadding
```

File: ui/shared/status/StatusGifPopup.qml [hunk 2]
```diff
@@ -102,23 +182,23 @@ Popup {
...
 
                 StatusGifColumn {
                     gifList.model: chatsModel.gif.columnA
-                    gifWidth: (popup.width / 3) - 12
+                    gifWidth: (popup.width / 3) - 16
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifPopup.qml:189]:
```suggestion
                    gifWidth: (popup.width / 3) - Style.current.padding
```

File: ui/shared/status/StatusGifPopup.qml [hunk 3]
```diff
@@ -102,23 +182,23 @@ Popup {
...
                     gifSelected: popup.gifSelected
                 }
 
                 StatusGifColumn {
                     gifList.model: chatsModel.gif.columnB
-                    gifWidth: (popup.width / 3) - 12
+                    gifWidth: (popup.width / 3) - 16
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifPopup.qml:195]:
```suggestion
                    gifWidth: (popup.width / 3) - Style.current.padding
```

File: ui/shared/status/StatusGifPopup.qml [hunk 4]
```diff
@@ -102,23 +182,23 @@ Popup {
...
                     gifSelected: popup.gifSelected
                 }
 
                 StatusGifColumn {
                     gifList.model: chatsModel.gif.columnC
-                    gifWidth: (popup.width / 3) - 12
+                    gifWidth: (popup.width / 3) - 16
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifPopup.qml:201]:
```suggestion
                    gifWidth: (popup.width / 3) - Style.current.padding
```

File: ui/shared/status/StatusGifPopup.qml [hunk 5]
```diff
@@ -71,19 +82,88 @@ Popup {
...
+                anchors.horizontalCenter: parent.horizontalCenter
+                source: "./assets/img/gif.svg"
+            }
+            
+            StyledText {
+                id: title
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: qsTr("Enable Tenor GIFs?")
+                font.weight: Font.Medium
+                font.pixelSize: 15
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifPopup.qml:141]:
```suggestion
                font.pixelSize: Style.current.primaryTextFontSize
```

File: ui/shared/status/StatusGifPopup.qml [hunk 6]
```diff
@@ -71,19 +82,88 @@ Popup {
...
+
+            StyledText {
+                id: headline
+                anchors.horizontalCenter: parent.horizontalCenter
+                text: qsTr("Once enabled, GIFs posted in the chat may share your metadata with Tenor.")
+                width: parent.width
+                horizontalAlignment: Text.AlignHCenter
+                wrapMode: Text.WordWrap
+                font.pixelSize: 13
+                color: Style.current.darkGrey
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifPopup.qml:152]:
```suggestion
                color: Style.current.secondaryText
```

File: ui/shared/status/StatusGifColumn.qml [hunk 7]
```diff
@@ -14,14 +16,36 @@ Column {
...
+            Rectangle {
+                id: loader
+                height: 100
+                width: root.gifWidth
+                visible: animation.status != Image.Ready
+                radius: Style.current.radius
+                rotation: 90
+                gradient: Gradient {
+                    GradientStop { position: 0.0; color: "transparent" }
+                    GradientStop { position: 1.0; color: "#E6EEF2" }
```
Comment (richard-ramos) [inline ui/shared/status/StatusGifColumn.qml:31]: Does this gradient color look okay in Dark Mode?

**Review by richard-ramos — CHANGES_REQUESTED** [review]
Nice work! added some minor observations

File: ui/shared/status/StatusGifColumn.qml (hunk 7 above)
Comment (alaibe) [reply ui/shared/status/StatusGifColumn.qml:33]: With a few needed adjustement, now it does work

**Review by richard-ramos — APPROVED** [review]

**Pull request merged**